sdk/javascript/package.json

```diff
@@ -13,8 +14,8 @@
     "test:jenkins": "c8 --require mocha --no-clean  --reporter=lcov npm run test",
     "vectors:jenkins": "c8 --require mocha --no-clean  --reporter=lcov npm run vectors",
     "catvectors:jenkins": "c8 --require mocha --no-clean  --reporter=lcov npm run catvectors",
-    "bundle": "webpack",
-    "bundle:clean": "rimraf ./_build"
+    "bundle": "mkdir -p ./dist && webpack",
```

Comment:
that mkdir _might_ be an issue on win, although not sure if we need to care
(it should work on win10+ I believe)